packages/openapi-generator/src/parser/operation-info.ts

```diff
@@ -0,0 +1,9 @@
+import { OpenAPIV3 } from 'openapi-types';
+import { OperationNameExtended } from '../extensions';
+import { Method } from '../openapi-types';
+
+export interface OperationInfo {
+  pathPattern: string;
```

Comment:
I could not find out what the `pathPattern` is. Could you add a doc with perhaps an example. Also the whole operation-info.ts feels a bit detached. So I would perhaps call it operation-types and include all operation related types?

Reply:
I guess this is just the path like `/some-service/get-somthing/{id}`.
I guess it's named like this because the openapi type call it pattern
```
    interface PathsObject<T extends {} = {}> {
        [pattern: string]: PathItemObject<T> | undefined;
    }
```

Reply:
Yes and also because it contains a pattern ;) I saw that @jjtang1985 used `path` in some other places. I am now using `pathPattern` consistently everywhere, but I don't have a strong opinion on this, we can also call it `path`. Do you have preferences?

@FrankEssenberger I updated the documentation in the mean time, if it is better now. I think `operation-types` does not really reflect what was meant.

Reply:
Discussed, that pathPattern is legit.